Server/ChatService/main.py

```
@@ -67,6 +67,31 @@
+    @app.get("/stats/websockets")
+    async def get_websocket_stats():
+        """Get WebSocket connection statistics"""
+        try:
+            if ws_manager:
+                return {
+                    "total_connections": ws_manager.get_total_connections(),
+                    "connected_users_count": len(ws_manager.get_connected_users()),
+                    "connected_users": ws_manager.get_connected_users(),
+                    "pubsub": ws_manager.get_pubsub_stats()
+                }
+            return {"error": "WebSocket manager not initialized"}
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)